src/content/resources/architectural-overview.md

```diff
@@ -1056,16 +1056,64 @@ is designed to interface with the
 underlying operating system rather than a web browser.
 A different approach is therefore required.
 
-On the web, Flutter offers two build modes and two renderers.
-Flutter has a canvas-based renderer with two compile modes:
-JS and Wasm.
-
-Flutter chooses the build mode when building the app,
-and determines which renderers are available at runtime.
-For a default build, Flutter chooses the `canvaskit`
-renderer at runtime. For a WebAssembly build,
-Flutter chooses the `skwasm` renderer at runtime,
-and falls back to canvaskit if the browser doesn't support skwasm.
+On the web, Flutter offers two renderers:
+
+<table class="table table-striped">
+<tr>
+<th>Engine</th>
+<th>Renderer</th>
+<th>Compilation target</th>
+</tr>
+
+<tr>
+<td>Engine 1</td>
```

Comment:
I only used the "engine 1" and "engine 2" to explain the situation for us here. We don't want to say that we have multiple engine. Multiple "renderers" is probably a better way to explain it. Can we use these tables instead?

Renderers:

| Renderer | Compilation target |
| ------- | ------- |
| CanvasKit | JavaScript |
| Skwasm | WebAssembly |

Build modes:

| Build mode | Available renderers |
| ------- | ------- |
| default | CanvasKit |
| `--wasm` | Skwasm (preferred), CanvasKit (fallback) |